api/presenters/PostPresenter.js

```diff
@@ -1,24 +1,41 @@
 const userColumns = q => q.column('users.id', 'users.name', 'users.avatar_url')
 
-var postRelations = (userId, opts) => _.filter([
-  {user: userColumns},
-  {communities: qb => qb.column('community.id', 'name', 'slug', 'avatar_url', 'banner_url')},
-  'contributions',
-  {'contributions.user': userColumns},
-  {followers: userColumns},
-  'media',
-  {relatedUsers: userColumns},
-  {responders: qb => qb.column('users.id', 'name', 'avatar_url', 'event_responses.response')},
-  (opts && opts.fromProject ? null : {projects: qb => qb.column('projects.id', 'title', 'slug')}),
-  {votes: qb => { // only the user's own vote
-    qb.column('id', 'post_id')
-    qb.where('user_id', userId)
-  }}
-], x => !!x)
+var postRelations = (userId, opts = {}) => {
+  var relations = _.filter([
+    {user: userColumns},
+    {communities: qb => qb.column('community.id', 'name', 'slug', 'avatar_url')},
+    'contributions',
+    {'contributions.user': userColumns},
+    {followers: userColumns},
+    'media',
+    {relatedUsers: userColumns},
+    {responders: qb => qb.column('users.id', 'name', 'avatar_url', 'event_responses.response')},
+    (opts && opts.fromProject ? null : {projects: qb => qb.column('projects.id', 'title', 'slug')}),
+    (opts && opts.withComments
+      ? {comments: qb => qb.column('comment.id', 'text', 'created_at', 'user_id', 'post_id')}
```

Comment:
ultimately this will return just the first 3 comments, but I thought I'd keep it like this for now for demoing the 'show all' on posts in the simplest way possible.